MIGRATION.md

```diff
@@ -82,7 +83,6 @@ The table below contains a list of `python-cloudant` functions and the `cloudant
 |`db.get_security_document()/db.security_document()`|[getSecurity](https://cloud.ibm.com/apidocs/cloudant?code=python#getsecurity)|
 |`db.share_database()`|[putSecurity](https://cloud.ibm.com/apidocs/cloudant?code=python#putsecurity)|
 |`db.shards()`|[getShardsInformation](https://cloud.ibm.com/apidocs/cloudant?code=python#getshardsinformation)|
-|`Document(db, 'docid').delete()`|[getLocalDocument](https://cloud.ibm.com/apidocs/cloudant?code=python#getlocaldocument)|
```

Comment:
This should be a `deleteDocument`.

Reply:
@vmatyusGitHub I moved this above so the local operations are grouped together: https://github.com/cloudant/python-cloudant/pull/489/files#diff-c1603ee6a674bb75255aa448057630506153e363297fd58668e6679f10909bc6R75

Reply:
Yes, I see that, but please note that a simple document delete is missing from this table: https://cloud.ibm.com/apidocs/cloudant?code=python#deletedocument

Reply:
ah yes, I'll add that!

Reply:
Added in https://github.com/cloudant/python-cloudant/pull/489/commits/2baf07d4878c09b2e329e101056888ae22c54e1b